doc/integrations/duckdb.md

```diff
@@ -15,6 +15,7 @@ myst:
     description lang=en: Use DuckDB from Jupyter using JupySQL
     keywords: jupyter, sql, jupysql, duckdb, plotting
     property=og:locale: en_US
+    property=og:image: (/_static/square-no-bg-small.png)
```

Comment:
I'm not sure it's the right syntax since this eventually becomes

```
 <meta property="og:image" content="https://jupysql--685.org.readthedocs.build(/_static/square-no-bg-small.png)">
```